pom.xml

```diff
@@ -327,6 +327,13 @@
             </plugin>
         </plugins>
     </build>
+    <repositories>
```

Comment:
nit: is the addition of `repositories` necessary for this change? Asking cause I don't see it in https://github.com/awslabs/deequ/pull/545